### PR TITLE
Fix byog QRNG

### PR DESCRIPTION
**Pull request by bbenligiray**

### CODE REVIEW

File: docs/qrng/reference/providers.md [hunk 1]
```diff
@@ -71,10 +78,16 @@ Australia. Visit their website at
...
 
-## BYOG Random Numbers
+## byog Random Numbers
+
+[byog](https://byog.io/) is an independent group within the API3 ecosystem that
+builds high-impact oracle services.
+
+::: warning Supported chains
+
+byog Random Numbers emulates the QRNG service on [testnets](./chains.md) using [pseudorandom number generation](../README.md).
```
Comment (bbenligiray) [inline docs/qrng/reference/providers.md:88]: Otherwise it might look deceptive

File: docs/qrng/reference/providers.md [hunk 2]
```diff
@@ -39,6 +39,13 @@ Australia. Visit their website at
 
 <!-- Need css for mobile -->
 
+::: warning Supported chains
+
+ANU Quantum Random Numbers is only available on [mainnets](./chains.md).
```
Comment (bbenligiray) [inline docs/qrng/reference/providers.md:44]: Without this you're asking for the user to try to use it on testnets

Comment (wkande) [reply docs/qrng/reference/providers.md:44]: So this is a change in that ANU cannot be used on a testnet any longer? So if a user wants to try QRNG they must use a byyog contract address only on testnet.

Can byog use main net?

Comment (bbenligiray) [reply docs/qrng/reference/providers.md:44]: Previously
ANU: all mainnets and goerli
Now
ANU: all mainnets (and no testnets)
byog: all testnets (and no mainnets)

**Review by metobom — APPROVED** [review]

**Comment by wkande** [comment]
In addition to these changes I will follow-up with a very brief note (of the same) on the `chains.md` page.

**Pull request merged**